doc/languages-frameworks/python.md

```diff
@@ -546,6 +546,26 @@ All parameters from `mkDerivation` function are still supported.
 * `catchConflicts` If `true`, abort package build if a package name appears more than once in dependency tree. Default is `true`.
 * `checkInputs` Dependencies needed for running the `checkPhase`. These are added to `buildInputs` when `doCheck = true`.
 
+##### Overriding Python packages
+
+The `buildPythonPackage` function has a `overridePythonPackage` attribute that
+can be used to override the package. The following example renamed the Pandas
+package to `foo`. We override first the Python interpreter and pass
+`packageOverrides` which contains the overrides for packages in the package set.
+
+```nix
+with import <nixpkgs> {};
+
+(let
+  python = let
+    packageOverrides = self: super: {
+      pandas = super.pandas.overridePythonPackage(old: {name="foo";});
```

Comment:
I can use `.overridePythonPackage` inside say `python-packages.nix` but not in a separate `shell.nix`.